_docs-sources/pipelines/supported-ci-systems/index.md

```diff
@@ -0,0 +1,5 @@
+# Supported CI systems
+
+Today, Gruntwork Pipelines works exclusively with [GitHub Actions](https://docs.github.com/en/actions).
+
+However, the [Gruntwork Pipelines architecture](../how-it-works) is designed to be agnostic to the underlying CI system, so we will add support for other CI systems in the future.
```

Comment:
This isn't technically true

Reply:
the `pipelines` binary _is_ ci/cloud agnostic, but that currently only contains about 30-40% of the total logic for what is the current iteration of the pipelines product. I'd remove this for now and simple state that we currently support github actions, if you want another CI system, reach out to the product team.